funding/manager.go

```diff
@@ -736,6 +736,76 @@ func (m *Manager) BatchChannelSetup(
 	)
 }
 
+// SidecarBatchChannelSetup will attempt to establish new funding flows with all
+// matched takers (people buying our channels) in the passed batch. This method
+// will block until the channel is considered pending. Once this phase is
+// complete, and the batch execution transaction broadcast, the channel will be
+// finalized and locked in.
+func (m *Manager) SidecarBatchChannelSetup(batch *order.Batch,
+	chanUpdates *subscribe.Client,
+	getOrder order.Fetcher) (map[wire.OutPoint]*chaninfo.ChannelInfo,
+	error) {
+
+	var (
+		chanPoints     = make(map[wire.OutPoint]order.Nonce)
+		fundingRejects = make(map[order.Nonce]*auctioneerrpc.OrderReject)
+	)
+
+	// We need to make sure our whole process doesn't take too long overall
+	// so we create a context that is valid for the whole funding step and
+	// use that everywhere.
+	setupCtx, cancel := context.WithTimeout(
+		context.Background(), m.cfg.BatchStepTimeout,
```

Comment:
Just leaving a note here, but curious to see if we'll need to either increase this value, or use a special value for sidecars particularly if we intend for this to be used in a mobile setting....